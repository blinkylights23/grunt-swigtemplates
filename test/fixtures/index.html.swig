{{ local }}
{{ globalInherit }}
{{ globalOverride }}
{{ globalInherit|racecar }}
{{ testMe() }}
{{ reverse('Racecar!') }}
{{ __('Hi!') }}
