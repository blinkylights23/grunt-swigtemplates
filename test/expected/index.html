Index
Inherited
Overridden
racecar
Hello from testMe!
!racecaR
Hi!
